### PR TITLE
Pfe

### PR DESCRIPTION
**Pull request by kylebuch8**
Switching elements from the RHElement and rh- namespace to PFElement and pfe- namespace.

### CODE REVIEW

File: .storybook/config.js [hunk 1]
```diff
@@ -3,7 +3,7 @@ import { setOptions } from "@storybook/addon-options";
 const req = require.context("../elements", true, /\.story\.js$/);
 
 setOptions({
-  name: "RHElements",
+  name: "PFElements",
```
Comment (castastrophe) [inline .storybook/config.js]: PF Elements or PFElements? Space or no-space?

**Comment by mwcz** [comment]
I'm not able to `npm install`, it fails with:

```
lerna ERR! npm install exited 1 in '@rhelements/rhelements'  
lerna ERR! npm install stderr:                               
npm ERR! code E404                                           
npm ERR! 404 Not Found: @pfelements/pfe-sass@*               
```

It seems like `@pfelements/pfe-sass` can't be found on npm, but I wouldn't have expected that to be a problem because the local pfe-sass should be linked up and used instead.

File: .storybook/config.js (hunk 1 above)
Comment (kylebuch8) [reply .storybook/config.js]: It should be PatternFly Elements.

Comment (kylebuch8) [reply .storybook/config.js]: I fix it.

**Comment by mwcz** [comment]
Here goes something.

**Pull request merged**